components/locid/Cargo.toml

```diff
@@ -35,13 +35,14 @@ all-features = true
 
 [dependencies]
 tinystr = { version = "0.4.10", features = ["alloc"], default-features = false }
+tinystr-neo = { path = "../../experimental/tinystr_neo" }
 serde = { version = "1.0", optional = true, default-features = false, features = ["alloc"] }
 writeable = { version = "0.2.1", path = "../../utils/writeable" }
 displaydoc = { version = "0.2.3", default-features = false }
 
 [dev-dependencies]
 criterion = "0.3.3"
-icu = { path = "../icu", default-features = false }
+# icu = { path = "../icu", default-features = false }
```

Comment:
FYI: This is a temporary change to make `cargo criterion` build; otherwise, `locale_canonicalizer` gets built, which breaks because we haven't migrated it to TinyAsciiStr yet.